app/src/main/java/org/p2p/wallet/common/ui/widget/ActionButtonsView.kt

```diff
@@ -19,7 +19,11 @@ class ActionButtonsView @JvmOverloads constructor(
 ) : FrameLayout(context, attrs, defStyleAttr) {
 
     enum class ActionButton {
-        BUY_BUTTON, RECEIVE_BUTTON, SEND_BUTTON, SWAP_BUTTON
+        BUY_BUTTON,
+        RECEIVE_BUTTON,
+        SEND_BUTTON,
+        SWAP_BUTTON,
+        SELL_BUTTON
```

Comment:
We don't have `SELL_BUTTON` in the action view, don't we?

Reply:
we do have now :)